packages/components/src/rating/stories/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import Rating from '../';
+import '../style.scss';
+
+export default {
+	title: 'WooCommerce Admin/components/Rating',
```

Comment:
Don't necessarily need a change here, just curious why the top level "WooCommerce Admin" is needed or if this is just a common structure we're using across all repos.

Reply:
I'm following the pattern we're using in blocks - I think it's good to represent the "namespace" where components come from, and any hierarchy. If we combine our storybooks in future this would be even more useful :)